src/_TextInputWrapper.tsx

```diff
@@ -104,6 +105,12 @@ export const TextInputBaseWrapper = styled.span<StyledBaseWrapperProps>`
       background-color: ${get('colors.input.disabledBg')};
       border-color: ${get('colors.border.default')};
     `}
+
+    ${props =>
+    props.monospace &&
+    css`
+      font-family: ${get('fonts.mono')};
```

Comment:
do we need to tweak the size here too? the new type face looks a bit larger than usual inputs so may look odd adjacent to them. May just be the fixed-pitch of the the monospace font playing tricks on my eyes though. cc. @mperrotti 
![Screenshot 2022-03-08 at 12 43 37](https://user-images.githubusercontent.com/13340707/157241101-554d1201-aeca-4a7a-b0a7-ddd8957bafd2.png)

Reply:
Yeah I feel the size issue too. Would appreciate help from @mperrotti on this design.

Reply:
Took another detailed look and it may just be how monospace fonts look. Its the same size as the one on the developer settings > PAT generator page. I think I'll go ahead with what we have.